app/services/invoices/payments/gocardless_service.rb

```diff
@@ -56,7 +58,10 @@ def update_payment_status(provider_payment_id:, status:)
         payment.update!(status: status)
 
         invoice_payment_status = invoice_payment_status(status)
-        payment.invoice.update!(payment_status: invoice_payment_status)
+        payment.invoice.update!(
+          payment_status: invoice_payment_status,
+          ready_for_payment_processing: invoice_payment_status != 'succeeded',
```

Comment:
`failed` is also considered as ready_for_payment_processing, then?

Reply:
yep:)